reduxfx-view/src/main/java/com/netopyr/reduxfx/vscenegraph/impl/patcher/NodeUtilities.java

```diff
@@ -191,7 +190,7 @@ public static Object getProperty(Object obj, String name) {
                                 .computeIfAbsent("dialog", key -> new HashMap<>())
                 ),
 
-                Case($(), Collections.emptyMap())
+                Case($(), new HashMap<>())
```

Comment:
Can we create a subclass of `TreeItem` that has a property-map like `Node` and use this subclass instead of the actual TreeItem class in the SceneGraph? The user of the VNode-TreeItem may not even notice that a subclass of TreeItem is used in the SceneGraph.